__tests__/commands/info.js

```diff
@@ -42,16 +42,18 @@ const expectedKeys = [
   'bugs',
   'license',
   'scripts',
-  'dependencies',
-  'devDependencies',
   'dist',
   'directories',
 ];
 
+// yarn now ships as built, single JS files so it has no dependencies
```

Comment:
It only started to fail after you've published that version since these tests were doing an `info` call on `yarn` which was published. Not really your fault :)

We should probably do those tests offline.